### DISCUSSION TITLE
Welcome to AxialAerospace Discussions!

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [zer0Kerbal/AxialAerospace#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by FuryFire2004** [comment]
hello, so are you merging all of axial aero into this mod, or is it just a common config library?

**Reply by zer0Kerbal** [reply]
well, it is a parts pack - but there will be very little here because locally am using symbolic folder links to put it all together and package it; every updated to one of the five other mods (Dreamer, WhimChaser, LanderTek, Axial Aerospace Ltd) will trigger a new release. Don't really need it for CKAN - but for SpaceDock and CurseForge (where you download and install) it will be a nice little helper to the end users.

**Reply by FuryFire2004** [reply]
ok!
